### PR TITLE
f-breadcrumbs@2.1.0 - Render <span> not <a> if no URL is given

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Changed
- If no URL is given, render `<span>` instead of `<a>` or `<router-link>`.

### Added
- Unit test coverage

### CODE REVIEW

**Review by kevinrodrigues — APPROVED** [review]

**Review by AndyMayJE — APPROVED** [review]

File: packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue [hunk 1]
```diff
@@ -23,14 +23,22 @@
...
-                        v-else
+                        v-else-if="url"
                         :href="url"
                         :class="[
                             $style['c-breadcrumbs-link'],
                             linkActiveClass(index)
                         ]">
                         {{ name }}
                     </a>
+                    <span
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue:34]: Out of interest – does this need to be in a `span` at all? Could you not add the default styles to the `li` and then just add the link tag if a URL is present?

Comment (xander-marjoram) [reply packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue:34]: Potentially not, although I think it makes the code a little more readable and sits nicely with the if/else-if/else pattern.

Comment (DamianMullins) [reply packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue:34]: You could use `template` in place of the `span` if you want to keep the if/else pattern?

Comment (ashleynolan) [reply packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue:34]: Yeh, that's what I was thinking. For me, markup should only be there if it's needed semantically or for hooking styles onto, so I'd look to remove this for a `template` tag if possible.

Comment (xander-marjoram) [reply packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue:34]: I'll test it out

**Pull request merged**